se/se_epub_generate_toc.py

```diff
@@ -128,7 +130,32 @@ def landmark_link(self, work_type: str = "fiction", work_title: str = "WORK_TITL
 
 		return out_string
 
-def get_file_epub_type(soup: BeautifulSoup) -> str:
+
+def get_first(node: EasyXmlTree, wanted: str):
```

Comment:
Is `get_first()` necessary? You can get the first occurrence using xpath for example: `//li[1]` returns the first `li` encountered.